azure_sdk/lib/azure_sdk.rb

```diff
@@ -6,3 +6,4 @@
 require 'v2017_03_09/v2017_03_09_profile_client'
 require 'latest/latest_profile_client'
 require 'v2018_03_01/v2018_03_01_profile_client'
+require 'v2019_03_01/v2019_03_01_profile_client'
```

Comment:
Soon we will have profiles for government clouds which will be called with -Profile rather than -Hybrid. The Hybrid profiles are for AZURESTACK and for all the public/gov clouds. Meaning the app developed with -Hybrid will work across all the clouds AZURESTACK, UsGov, AzureChina and AzureGermany..

I am thinking whether we should call the client as v2019_03_01_hybrid instead of v2019_03_01_profile.. I am checking with Vishnu on the other SDKs usage pattern. I understand we will deviate from the 2018 profile..

Reply:
We have the _hybrid for Java/C#/Python SDKs, but not for Go and Ruby. @sarangan12 going forward can we call this with _hybrid

Reply:
Done